roles/StackStorm.nodejs/defaults/main.yml

```diff
@@ -1,3 +1,3 @@
 ---
 # nodejs version to install
-nodejs_major_version: "10"
+nodejs_major_version: "{% if st2chatops_version == 'latest' or st2chatops_version is version('3.5.0', '>=') %}14{% else %}10{% endif %}"
```

Comment:
What if the user sets the `st2chatops_version=present`?
This is possible for both scenarios when the older st2 version and newer st2 version is installed.

Reply:
In Ansible deployment we normally don't provide any migration, support, or handling for the previous st2 versions. And so it means we just support the latest st2 version, allowing users to pin the Ansible roles to the previous version if they want it to work with the older st2.

With that, I'd just recommend removing the conditional and set it to `14` by default.

Reply:
Thanks for the input. That's good to know and will be helpful for my future contributions. Did it as suggested.